src/spikeinterface/curation/tests/test_curation_format.py

```diff
@@ -152,10 +171,26 @@ def test_curation_label_to_dataframe():
     # print(df)
 
 
+def test_apply_curation():
+    recording, sorting = generate_ground_truth_recording(durations=[10.], num_units=9, seed=2205)
+    sorting._main_ids = np.array([1, 2, 3, 6, 10, 14, 20, 31, 42])
```

Comment:
you can use `sorting.rename_units`

Reply:
No I prefer this because then it is inplace.

Reply:
for testing this is OK

Reply:
speed for test. no duplication in this case.

The rename_units internally use UnitsSelectionSorting which is extremly unefficient because go back to dict of spiketrains and then construct again numpysorting at analyzer create step which trigger a full concatenate and lexsort.

I think this inplace main_ids setting  is OK because we get the idea.

Reply:
in this case it would be very quick